src/app/loaders/express/helmet.ts

```diff
@@ -54,6 +54,7 @@ const helmetMiddlewares = () => {
       "'self'",
       'https://www.google-analytics.com/',
       'https://ssl.google-analytics.com/',
+      'https://*.browser-intake-datadoghq.com',
```

Comment:
can link to the docs where they list this domain

Reply:
https://github.com/opengovsg/FormSG/pull/3887/commits/ae7345811979a47b4be5f05b0eb1305f044362c7